regex/rstring/string.go

```diff
@@ -41,3 +42,16 @@ func SplitString(n int, strs ...string) []string {
 
 	return split
 }
+
+// RemoveNonAlphaNumeric will remove any non-alphanumeric character from the input string
+// And return a newly processed string.
+func RemoveNonAlphaNumeric(input string) string {
+	pattern := "[^a-zA-Z0-9]+"
```

Comment:
@Norbinsh do we really need '+' in pattern ("[^a-zA-Z0-9]+"),  if we are replacing string

Reply:
* updated my fork with latest master
* removed repeating expression (plus sign) as it's not required - thanks.